bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYBBJetsToLL_M-50_5f_LO_MLM/DYBBJetsToLL_M-50_5f_LO_MLM_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+import model sm-ckm_no_b_mass
+
+define p = p b b~
+define j = p
+define bb = b b~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+
+# Specify process(es) to run
+add process p p > l+ l- bb bb / h @0
```

Comment:
usually preferred to have 
"generate" rather than "add process" for the first one
(I have seen this cause problems in the past at least)